harold/plugins/deploy.py

```diff
@@ -47,6 +47,36 @@ def __init__(self, http, monitor):
         self.monitor = monitor
 
 
+class DeployGetSalonInfoListener(DeployListener):
+    isLeaf = True
+
+    def _handle_request(self, request):
+        salons = yield self.salons.all()
+
+        salons_by_name = {}
+        for salon in salons:
+            salon_info = {
+                'name': salon.name,
+                'channel': salon.channel,
+                'allow_deploys': salon.allow_deploys,
```

Comment:
Per slack conversation, let's only expose the data needed to support the current use case.

Reply:
Done.